frontend/src/components/pages/EditTeamPage.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useContext, useState } from "react";
+import SampleContext from "../../contexts/SampleContext";
+import MainPageButton from "../common/MainPageButton";
+import SampleContextDispatcherContext from "../../contexts/SampleContextDispatcherContext";
+
+type DeleteButtonProps = { index: number; onClick: (index: number) => void };
+
+const DeleteMemberButton = ({ index, onClick }: DeleteButtonProps) => {
+  return (
+    <button
+      type="button"
+      onClick={() => {
+        onClick(index);
+      }}
+    >
+      Delete
+    </button>
+  );
+};
+
+const EditTeamInfoPage = () => {
+  const { teamName, numTerms, members } = useContext(SampleContext);
+  const dispatchTeamUpdate = useContext(SampleContextDispatcherContext);
+  const [newMember, setNewMember] = useState<string>("");
+
+  const setName = (e: React.FormEvent<HTMLInputElement>) => {
+    dispatchTeamUpdate({ type: "EDIT_NAME", value: e.currentTarget.value });
+  };
+
+  const setNumTerms = (e: React.FormEvent<HTMLInputElement>) => {
+    dispatchTeamUpdate({
+      type: "EDIT_NUM_TERMS",
+      value: parseInt(e.currentTarget.value, 10),
+    });
+  };
```

Comment:
State isn't getting updated for some reason (stuck at 3), unsure why 🤔

Reply:
Looks like it's just a typo in the reducer, see comments in `frontend/src/reducers/SampleContextReducer.ts`